articles/introduction-to-avl-trees/index.md

```diff
@@ -0,0 +1,548 @@
+### Introduction to AVL trees
+Binary search trees are one of the most efficient data structures with their O(n) time complexity while performing operations. But in this fast paced world, especially in the realm of computers and processing, the one parameter that everything is scrutinized by is speed. The AVL tree, named after its two inventors, G.M. Abelson-Velvety and E.M. Landis, who published it in their 1962 paper "An Algorithm For The Organization Of Information" has anchored its position as a need-to-understand data structure owing to its performance increase from a regular BST.
+
```

Comment:
Your article heavily relies on the reader's knowledge of Binary Trees and Binary Search Trees. Therefore, please add a prerequisites section in your article that highlights the requirements to read this article. 

Also, add a couple of links that would help the reader get to speed with the concepts related to BSTs.

Reply:
Revisions made in the recent commit.